main.js

```diff
@@ -0,0 +1,170 @@
+const apiKey = "9bd6ab08eafd640bd15b662631d477b6";
+var currWeather = [];
```

Comment:
var is pre-ES6 way to declare variables. let/const is ES6, please stay consistent in declaring variables.